packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Properties/launchSettings.json

```diff
@@ -90,11 +90,6 @@
       "commandName": "Executable",
       "executablePath": "$(SolutionDir)/../dist/generator/Microsoft.Generator.CSharp.exe"
     },
-    "http-parameters-spread": {
```

Comment:
What happened to these?

Reply:
See these issues:
https://github.com/microsoft/typespec/issues/5568
https://github.com/microsoft/typespec/issues/5567
https://github.com/microsoft/typespec/issues/5565

Reply:
These are fixed.